### DISCUSSION TITLE
Welcome to the Phaser Editor 2D forum!

### DISCUSSION THREAD

**Discussion by PhaserEditor2D** [PhaserEditor2D/PhaserEditor2D-v3#94] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by BootsManOut** [comment]
Hello there!
I thought Phaser Editor looked super fun! I enjoy creating games in my free time!
So here I am looking for help while I start out with Phase Editor 2D!

**Reply by PhaserEditor2D** [reply]
Hi! Welcome to our community!

I'm here for supporting you on any issue you have with the editor!

Also, I invite you to join us in our Discord server: https://discord.com/invite/4DdpMMD
